pkg/files/file_helpers.go

```diff
@@ -58,21 +59,26 @@ func GenerateConfigVersion(fileSlice []*v1.File) string {
 		hashes += file.GetFileMeta().GetHash()
 	}
 
-	return uuid.Generate("%s", hashes)
+	return GenerateHash([]byte(hashes))
 }
 
-// GenerateFileHash returns the hash value of a file's contents.
-func GenerateFileHash(filePath string) (string, error) {
-	f, err := os.Open(filePath)
-	if err != nil {
-		return "", err
+// GenerateHash returns the hash value of a file's contents.
+func GenerateHash(b []byte) string {
+	return uuid.NewMD5(uuid.Nil, b).String()
+}
+
+// ReadFile returns the content of a file
+func ReadFile(filePath string) ([]byte, error) {
```

Comment:
Just realised that we dont need this function. There is already a ReadFile function in the golang os library.